src/ElasticSearchable.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ScoutEngines\Elasticsearch;
+
+use Laravel\Scout\Builder as ScoutBuilder;
+use Laravel\Scout\Searchable;
+
+trait ElasticSearchable
```

Comment:
is it ok, that all mapping in this trait is overriding by toSearchableArray() method?